Conversations 3 7 (#98)
tyntec/api-collection: conversations/v3/README.md

# Release notes

## 3.7

### Features

> #### WhatsApp supporting Product and Product Lists

Reviewer: ```suggestion
#### WhatsApp support Product and Product Lists
```

> WhatsApp supports now sending products from a Facebook catalog.

Reviewer: ```suggestion
WhatsApp now supports sending products from the Facebook Catalog.
```

> This could be either a single product or multiple products via a product list.

Reviewer: ```suggestion
It can be either a single product or multiple products through the product list.
```

> The message to be used is of type `interactive` with the respective subtype `product` or `productList`.

Reviewer: ```suggestion
The message type is `interactive` with the subtype `product` or `productList`.
```

> The catalog must be prior connected to the WhatsApp Business Account that shall have access to it.

Reviewer: ```suggestion
You need to link your Facebook Catalog to your WhatsApp Business Account. 
```

> The customer can place an order via the app, in this case the new MoMessage event `MoMessage::Order` is triggered.

Reviewer: ```suggestion
The customer can place an order through their app. Then a new MoMessage event `MoMessage::Order` is triggered.
```
